docs/migration/dapp_migration.md

```diff
@@ -25,7 +25,7 @@ For starters, here's a sampling of exciting perks dapps with get with the Nitro
 ## Breaking changes
 
 #### Dapps
-- **Gas Accounting**: it is now consistent with the L1 EVM, any hard-coded gas values should be changed accordingly (the same applies to any gas amount used in conjuntion with `gasleft`).
+- **Gas Accounting**: it is now consistent with the L1 EVM, L2 gas usage will change due to different accounting from ArbGas. Any hard-coded gas values should be changed accordingly (the same applies to any gas amount used in conjuntion with `gasleft`).
```

Comment:
Not a comment on this PR, but instead the existing text. I think we should generally strongly warn people away from hard-coding gas values just like ethereum does since the L1 and L2 gas schedule may change in the future